json-endpoints/jetpack/class.jetpack-json-api-get-post-backup-endpoint.php

```diff
@@ -3,7 +3,7 @@
 class Jetpack_JSON_API_Get_Post_Backup_Endpoint extends Jetpack_JSON_API_Endpoint {
 	// /sites/%s/posts/%d/backup      -> $blog_id, $post_id
 
-	protected $needed_capabilities = 'read_private_posts';
+	protected $needed_capabilities = array();
```

Comment:
Clearing out the needed capabilities -- I assume this is so it can be done in a user-agnostic way -- or when there is no logged in user -- do we have anywhere we're using this now that might be a bit dangerous to remove the existing check?